src/libs/actions/IOU.ts

```diff
@@ -2075,9 +2075,15 @@ function getMoneyRequestInformation(
             : ReportUtils.buildOptimisticIOUReport(payeeAccountID, payerAccountID, amount, chatReport.reportID, currency);
     } else if (isPolicyExpenseChat) {
         iouReport = {...iouReport};
-        if (iouReport?.currency === currency && typeof iouReport.total === 'number') {
-            // Because of the Expense reports are stored as negative values, we subtract the total from the amount
-            iouReport.total -= amount;
+        // Because of the Expense reports are stored as negative values, we subtract the total from the amount
+        if (iouReport?.currency === currency) {
+            if (typeof iouReport.total === 'number') {
+                iouReport.total -= amount;
+            }
+
+            if (typeof iouReport.unheldTotal === 'number') {
+                iouReport.unheldTotal -= amount;
+            }
```

Comment:
Is there a case where `unheldTotal` wouldn't exist? If we have an expense report where all the transactions are reminbursable will we have:
1. `unheldTotal` set to `total`
2. `unheldTotal` not set at all (`undefined`)

cc @robertjchen

Reply:
In `main`, we don't add `unheldTotal` and `unheldNonReimbursableTotal` when creating the report optimistically (buildOptimisticExpenseReport only, IOU not yet, waiting for the IOU total bug to be fixed), so it will be undefined. But in this PR, I already added both optimistically, so it will never be undefined anymore optimistically. I don't know about the BE.

Reply:
For BE, we will always return these fields.

Reply:
Thanks for the confirmation. That means we could technically update the typing for the `total`, `unheldTotal`, `nonReimbursableTotal`, and `unheldNonReimbursableTotal` to be non-undefined/null, but if we want to do that, we need to update all report optimistic creation (buildOptimisticChatReport, buildOptimisticTaskReport, etc.) to include them all, which isn't the case for now.